subsys/bluetooth/host/conn.c

```diff
@@ -173,14 +173,13 @@ static void tx_notify(struct bt_conn *conn)
 		void *user_data;
 
 		key = irq_lock();
-		if (sys_slist_is_empty(&conn->tx_complete)) {
-			irq_unlock(key);
+		tx = CONTAINER_OF(sys_slist_get(&conn->tx_complete), struct bt_conn_tx, node);
```

Comment:
Right now this will work I think, but if in the future the offset is non-null, the if test below would break if `sys_slist_get` returns NULL.